tests/legacy/config.js

```diff
@@ -8,6 +8,14 @@ var compare = require('./compare');
 var webpackMajorVersion = require('webpack/package.json').version.split('.')[0];
 
 module.exports = function(OfflinePluginOptions, testFlags) {
+  testFlags = testFlags || {};
+
+  if (testFlags.minimumWebpackMajorVersion &&
+      parseInt(webpackMajorVersion, 10) < testFlags.minimumWebpackMajorVersion) {
+    throw new Error('unsupported webpack version');
+  }
+  delete testFlags.minimumWebpackMajorVersion;
```

Comment:
How come you are deleting this? If set, would you not expect it to be present further down the execution flow? Maybe I am missunderstanding something :)